app/scripts/modules/core/src/pipeline/config/stages/savePipelines/savePipelinesConfig.controller.ts

```diff
@@ -0,0 +1,38 @@
+import { IController, IScope } from 'angular';
+import { defaults } from 'lodash';
+
+export class SavePipelinesConfigCtrl implements IController {
```

Comment:
I think we should avoid net new angular code. Take a look at `EvaluateVariablesStageConfig` for an example of a React-based stage that doesn't require an angular bridge.

Reply:
I started with one like that as an example, but abandoned it because stage-artifact-selector-react wasn't working correctly in the react HTML, and the only example of using stage-artifact-selector-react is like this.

Reply:
yes, I'll update it to look like that